frontend/src/pages/Register/Register.jsx

```diff
@@ -1,18 +1,21 @@
 import { useState } from 'react';
+import Col from 'react-bootstrap/Col';
 import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+
 import { useParams } from 'react-router-dom';
 
 import EmailVerify from './EmailVerify';
 import styles from './Register.module.css';
+
 import RegisterForm from './RegisterForm';
 
 import { register } from '../../api';
-import FatherSonBlock from '../../components/FatherSonBlock';
+// import FatherSonBlock from '../../components/FatherSonBlock';
 import Header from '../../components/Header';
-import TextLink from '../../components/TextLink';
+// import TextLink from '../../components/TextLink';
```

Comment:
Will we need commented out code in the future?

Reply:
Hii all the commented lines from the code has been removed.